### PR TITLE
PR 

### PR DESCRIPTION
**Pull request by asiday**

### CODE REVIEW

**Comment by yambay** [comment]
Please squash before merge

File: crawler/src/main/java/dev/asida/crawler/configurations/RequestContext.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        return domains;
+    }
+    public List<Language> getLanguages() {return languages;}
+    public List<Category> getCategories() {
+        return categories;
+    }
+    public List<Country> getCountries() {
+        return countries;
+    }
+    public List<DateFrom> getDatesFrom() {return datesFrom;}
```
Comment (yambay) [inline crawler/src/main/java/dev/asida/crawler/configurations/RequestContext.java:30]: Why do we need a list of dates here? Maybe it should be a single date?

File: crawler/src/main/java/dev/asida/crawler/configurations/RequestContext.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    public List<DateTo> getDatesTo() {return datesTo;}
```
Comment (yambay) [inline crawler/src/main/java/dev/asida/crawler/configurations/RequestContext.java:31]: Same here: Maybe a single date would be better.

File: crawler/src/main/resources/application.properties [hunk 3]
```diff
@@ -1 +1 @@
-
+key=cdf6501914d546bfb64df09b1c6ebbb6
```
[the remark this answers is not in the document]
Comment (asiday) [reply crawler/src/main/resources/application.properties:1]: Key saved as environment variable.

File: pom.xml [hunk 4]
```diff
@@ -17,13 +17,24 @@
...
+        </repository>
+    </repositories>
     <dependencies>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>4.13.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
[the remark this answers is not in the document]
Comment (asiday) [reply pom.xml:33]: Removed

File: pom.xml [hunk 5]
```diff
@@ -17,13 +17,24 @@
     <properties>
         <java.version>11</java.version>
     </properties>
+    <repositories>
```
[the remark this answers is not in the document]
Comment (asiday) [reply pom.xml:20]: Deleted from main and from crowler. No need.

File: pom.xml [hunk 6]
```diff
@@ -17,13 +17,24 @@
...
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+    </repositories>
     <dependencies>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>4.13.2</version>
```
[the remark this answers is not in the document]
Comment (asiday) [reply pom.xml:30]: Using Hamcrest. I've deleted for now all junit dependecies.
Also deleted dependencies for the library  com.github.KwabenBerko and JitPack repository. 
            <

**Review by yambay — APPROVED** [review]

**Pull request merged**